http/src/config.rs

```diff
@@ -167,7 +197,7 @@ pub fn load(config: File) -> Result<(ipfs::Keypair, Vec<Multiaddr>), LoadingErro
         });
     }
 
-    Ok((kp, addresses.swarm))
+    Ok((kp, addresses.swarm, addresses.api))
```

Comment:
`CompatibleConfigFile` is probably currently private and that is the reason behind using tuples here along I guess? Tuples probably do not scale up to adding the bootstrap addresses so we'll need to remember to change this then at last.